flink/v1.18/flink/src/test/java/org/apache/iceberg/flink/source/TestFlinkMetaDataTable.java

```diff
@@ -164,21 +163,22 @@ public void testHistory() {
     Iterator<Snapshot> snapshots = table.snapshots().iterator();
     for (Row row : result) {
       Snapshot next = snapshots.next();
-      Assert.assertEquals(
-          "Should have expected made_current_at",
-          ((Instant) row.getField(0)).toEpochMilli(),
-          next.timestampMillis());
-      Assert.assertEquals("Should have expected snapshot id", next.snapshotId(), row.getField(1));
-      Assert.assertEquals("Should have expected parent id", next.parentId(), row.getField(2));
-
-      Assert.assertEquals(
-          "Should have expected is current ancestor",
-          SnapshotUtil.isAncestorOf(table, table.currentSnapshot().snapshotId(), next.snapshotId()),
-          row.getField(3));
+      assertThat(((Instant) row.getField(0)).toEpochMilli())
```

Comment:
I'm confused about what exactly is the actual vs expected. `testSnapshots()` has the same issue

Reply:
I fixed the order